src/main/java/codeu/model/store/persistence/PersistentDataStore.java

```diff
@@ -146,13 +151,28 @@ public PersistentDataStore() {
     return messages;
   }
 
-  /** Write a User object to the Datastore service. */
+  /** Write a new User object to the Datastore service. */
   public void writeThrough(User user) {
```

Comment:
Good catch on updating the comment - renaming this method to "createUser" may be more appropriate.